NEWS

```diff
@@ -11,12 +11,36 @@
 Features:
 - TBD
 
+## 1.6.0-rc1 (May 24, 2019)
+Features:
+- Modular architecture for UCT transports
+- ROCM transport re-design: support for managed memory, direct copy, ROCM GDR
+- Random scheduling policy for DC transport
+- Optimized out-of-box settings for multi-rail
+- Added support for OmniPath (using Verbs)
+- Support for PCI atomics with IB transports
+- Reduced UCP address size for homogeneous environments
+
+Bugfixes:
+- Multiple stability and performance improvements in  TCP transport
+- Multiple stability fixed in Verbs and MLX5 transports
+- Multiple stability fixes in UCM memory hooks
+- Multiple stability fixes in UGNI transport
+- RPM Spec file cleanup
+- Fixing compilation issues with most recent clang and gcc compilers
+
 Tested configurations:
 - RDMA: MLNX_OFED 4.5, distribution inbox drivers, rdma-core 22.1
-- CUDA: gdrcopy 1.3.2, cuda 9.2
+- CUDA: gdrcopy 1.3.2, cuda 9.2, ROCM 2.1
```

Comment:
ROCM -> ROCm
@yuq @bbenton is this the right version?

Reply:
ROCm 2.1 has a bug which can't support >4G buffer transfer, ROCm >=2.2 is fine.